lib/sync/fetcher_test.go

```diff
@@ -36,9 +35,9 @@ func TestBlockFetcher(t *testing.T) {
 	st := block.InitTestBlockchain()
 	defer st.Close()
 
-	kp := keypair.Random()
-	_, _, localNode := network.CreateMemoryNetwork(nil)
+	localNode := node.NewTestLocalNode0()
 
+	kp := keypair.Random()
```

Comment:
Why did you remove from above line and inserted again?

Reply:
I just grouped it with its usage (`NewValidator`). Because a local node also takes a keypair.